projects/table-builder/src/lib/components/header-menu/header-menu.component.ts

```diff
@@ -89,4 +89,17 @@ export class HeaderMenuComponent {
     }) );
   }
 
+  onEnter(filter: FilterInfo, event) {
+    console.log('object')
+    event.stopPropagation();

```

Comment:
can you use the ```stop-propagation``` directive directly in the template instead?

Reply:
I have pushed changes. I did not really need stop propagation. Please review.